src/ui/__fixtures__/filteredIdentifierFix.ts

```diff
@@ -54,6 +33,6 @@ const filteredKeriFix: IdentifierShortDetails[] = [
   },
 ];
 
-const filteredIdentifierFix = [...filteredDidFix, ...filteredKeriFix];
+const filteredIdentifierFix = [...filteredKeriFix];
 
-export { filteredDidFix, filteredKeriFix, filteredIdentifierFix };
+export { filteredKeriFix, filteredIdentifierFix };
```

Comment:
Just name the main one as `filteredIdentifierFix` and export one. We don't need another variable copying the contents of the first since there's only one now